pkgs/development/python-modules/pyvex/default.nix

```diff
@@ -35,5 +36,7 @@ buildPythonPackage rec {
     homepage = "https://github.com/angr/pyvex";
     license = with licenses; [ bsd2 gpl3Plus lgpl3Plus ];
     maintainers = with maintainers; [ fab ];
+    # ERROR: pyvex-X-py3-none-manylinux1_aarch64.whl is not a supported wheel on this platform.
+    broken = stdenv.isAarch64;
```

Comment:
interesting.. we build from source

Reply:
I'm no expert on building with `manylinux1` or `manylinux2014`. Thus, I don't really know where to start debugging this issue.